packages/tools/src/types/InterpolationTypes.ts

```diff
@@ -0,0 +1,18 @@
+import { Types } from '@cornerstonejs/core';
+import { Annotation } from './AnnotationTypes';
+import { InterpolationROIAnnotation } from './ToolSpecificAnnotationTypes';
+
+export type InterpolationViewportData = {
+  /** The annotation that is being updated with a change in label. */
+  annotation: InterpolationROIAnnotation;
+  /** The type of event, whether initializing the label or updating it. */
+  interpolationUID: string;
+  /** unique id of the viewport */
+  viewport: Types.IViewport;
+  sliceData: Types.ImageSliceData;
+};
+
+export type ImageInterpolationData = {
+  sliceIndex: number;
+  annotations?: Annotation[];
```

Comment:
Shouldn't this be InterpolationROIAnnotation[]?

Reply:
Yes, most likely it should but the casting isn't quite right to do that now.  We can talk about whether we want to fix that.